app/controllers/users/passwords_controller.rb

```diff
@@ -24,40 +27,39 @@ def update
       @password_form = PasswordForm.new(resource)
 
       if @password_form.submit(user_params)
-        handle_successful_password_reset_for(resource)
+        handle_successful_password_reset
       else
-        if resource.errors[:reset_password_token].present?
-          return handle_expired_reset_password_token
-        end
-
-        render :edit
+        handle_unsuccessful_password_reset
       end
     end
 
     protected
 
     def token_user
-      @token_user ||= User.with_reset_password_token(params[:reset_password_token])
+      @_token_user ||= User.with_reset_password_token(params[:reset_password_token])
```

Comment:
What is this trend for starting instance variables with "_"?   Should we apply it consistently and document it if there is some rule here?